docs/endpoints.rst

```diff
@@ -33,7 +33,9 @@ Aggregation Endpoints
 .. _time: time.html#time
 .. _filter: filter.html#filter
 
-**Example request**:
+**How big is the area of farmland in the region Rhein-Neckar?**
```

Comment:
It's one possibility to add a header. I, personally, would be in favour of a description below “Example Request”. This current test leads to unpleasant looking indentions ;)
![Clipboard - July 30, 2020 3_11 PM](https://user-images.githubusercontent.com/2878899/88927170-522f6380-d277-11ea-9118-e8da6e37b9a3.png)

Reply:
The indention was on purpose. I wanted to show, how it could look and how it looked. "Example response" should also be moved lightly to the right side.